components/ButtonBar/src/MDCButtonBar.h

```diff
@@ -111,6 +111,12 @@ IB_DESIGNABLE
  */
 - (CGSize)sizeThatFits:(CGSize)size;
 
+/**
+ The Material inkColor that is used for all buttons in the button bar. Only affects buttons that are
+ created after changing the property.
+ */
+@property(nonatomic, strong, nullable) UIColor *inkColor;
```

Comment:
Document that if this property is nil then the default button ink color will be used instead.

Reply:
Done.